wordpress-mobile/WordPress-iOS
# [Gutenberg] Fix crash when creating New Photo Post on gutenberg from 3D Touch shortcut

This PR implements the Gutenberg Bridge changes from: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1271

his PR fixes an issue that crashes the app when trying to start a photo post from the 3D Touch shortcuts.

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/1190

To test:
- Build and run on a iOS **device with 3D touch** (iPhone 6S family or later)
- When the app is running, send it to background
- Find the WordPress app icon, and "force touch" it to display the shortcut options.
- Select "New Photo Post"
- Choose an image (or a few)
- Press on "Insert" at the bottom right corner
- Check that the images are added to the post and there is no crash

Update release notes:
